english/DC-Comics/epic_DC_Character.md

```diff
@@ -6,3 +6,7 @@ Give it a deep thouht and figure it out.
 
 Hint: A useless character :)
 
+OR
+
+[Become SuperMe ...](english/DC-Comics/superMe/superMe.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.